### ISSUE TITLE
Should participants of the AC adhere to the vision of the project?

### ISSUE THREAD

**Issue by tobie** [ampproject/meta-ac#56]
In his review @triblondon asked the following question:

> Clarify whether participants in the AC should only be those who agree with the vision, or whether it can include those who don't. ie. does joining the AC or any other AMP group signal your agreement to the overall goals of the AMP project?

**Comment by LJWatson** [comment]
I'm not sure what this question is really asking.

If the question is literal, , I think we already have the answer in the 
form of people like @edent, @melsumner, and myself.

If the question is asking if we can clarify this in our documentation 
somewhere, then strong +1.

**Comment by tobie** [comment]
Yeah, I think this was about the governance document which [states](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#membership) that: 

> Membership on the Advisory Committee shall include representatives from major AMP constituencies (Collaborators, Contributors, Users and End-Users) who are committed to fulfilling AMP's vision and mission.

We discussed this at the [F2F in London](https://github.com/ampproject/meta-ac/blob/master/meetings/2019-03-f2f.md#membership-of-ac), came to the conclusion _"that the vision / mission needs works,"_ and filed https://github.com/ampproject/meta-ac/issues/31 to work on that.

**Comment by jorydotcom** [comment]
Can this issue be closed, @tobie?
